assets/localization-form.js

```diff
@@ -22,7 +22,9 @@ if (!customElements.get('localization-form')) {
     onContainerKeyUp(event) {
       if (event.code.toUpperCase() !== 'ESCAPE') return;
 
+      if(this.elements.button.getAttribute('aria-expanded') == 'false') return;
       this.hidePanel();
+      event.stopPropagation();
```

Comment:
Here I added some logic so that when pressing the `ESC` key, it doesn't trigger both the local dropdown being closed and the menu drawer. 
Now it should close the dropdown if it's open. Then if you press again it will close the menu drawer.